internal/cli/cmd/root.go

```diff
@@ -44,9 +56,42 @@ func Setup(ctx *cli.Context) error {
 	if ctx.Bool("debug") {
 		s.Logger().SetLevel(log.DebugLevel())
 	}
+
+	if err = SetLoggerTarget(s, ctx); err != nil {
+		return err
+	}
+
+	ctx.App.After = Teardown
+
 	if ctx.App.Metadata == nil {
 		ctx.App.Metadata = map[string]any{}
 	}
 	ctx.App.Metadata["system"] = s
 	return nil
 }
+
+func Teardown(ctx *cli.Context) error {
+	return logFile.Close()
```

Comment:
This is going to panic with nil dereference in the cases where we don't log to file. Perhaps a single `if logFile != nil` first should be enough?